tests/test_idpi/test_diff.py

```diff
@@ -0,0 +1,43 @@
+# Third-party
+import numpy as np
+from numpy.testing import assert_allclose
+
+# First-party
+from idpi import grib_decoder
+from idpi.operators import diff
+from idpi.operators.theta import ftheta
+
+
+def test_padded_field(data_dir, grib_defs):
```

Comment:
there is not anymore any padded field in this PR. I guess the name is based on the previous implementation, so should we rename the test?